com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -520,6 +520,17 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
             if (useRenderPassEnabled || useDepthPriming)
                 createColorTexture |= createDepthTexture;
 
+            //Set rt descriptors so preview camera's have access should it be needed
```

Comment:
This comment seems obsolete now.

Reply:
I put it there so it's clear why we set the descriptor there even when we are not sure an intermediate RT is needed.